composer.json

```diff
@@ -31,7 +31,11 @@
         "php": "^7.2 || ^8.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "^8.0 || ~9.4.4"
+        "phpunit/phpunit": "^8.0 || ~9.4.4",
+        "friendsofphp/php-cs-fixer": "~3.2.0 || ^3.13"
+    },
+    "scripts": {
+        "php-cs-fixer": "vendor/bin/php-cs-fixer fix --diff --using-cache=yes"
```

Comment:
Since linters workflow was added, it makes sense to add a dependecy on php-cs-fixer so we can easy run linters on local

Reply:
I followed up with this: https://github.com/predis/predis/commit/f6f4830b3c815cca30bc56faf23f21046304531a